### DISCUSSION TITLE
x86 wheels are built for macOS 11.0 when building with `CIBW_ARCHS: x86_64 arm64`

### DISCUSSION THREAD

**Discussion by nicholasjng** [pypa/cibuildwheel#913] [Q&A] [answered]
Hello `cibuildwheel` community,

I am trying to build macOS wheels of a package for both macOS x86 and ARM64, using the `CIBW_ARCHS` option. I run on GitHub Actions' `macos-10.15`, and my subsequent wheel testing fails because a `macosx_11_0` wheel is built for `x86` (which is unsupported). 

Correct me if I am wrong, but shouldn't the `MACOSX_DEPLOYMENT_TARGET` still be "10.9" for the `x86` wheels, even when building `arm64` wheels at the same time? I also explicitly set the target to `10.9`, but it still ends up building macOS 11 wheels. I was able to find some issues on that environment variable, but nothing that looked quite like the problem here.

I think I am missing something about the macOS deployment configuration here, and wanted to get some direction before opening an issue. Thank you for your consideration.

**Comment by henryiii** [comment, marked as answer]
Do you have a link to the package? Sounds like possibly an issue with your setup. Normal setuptools, scikit-build, other?

**Reply by nicholasjng** [reply]
It's Google benchmark, where I'm currently contributing wheel CI in my free time. I am testing workflow setups here on the `wheel_action` branch in my fork:

https://github.com/nicholasjng/benchmark/tree/wheel-action

Here is the `setup.py`, using setuptools:

https://github.com/nicholasjng/benchmark/blob/wheel-action/setup.py

**Reply by henryiii** [reply]
It's a Bazel wrapper. Going to bet they didn't add the Bazel equivalent of https://github.com/pybind/cmake_example/blob/2440893c60ed2578fb127dc16e6b348fa0be92c1/setup.py#L94-L98 .

**Reply by nicholasjng** [reply]
Right on the money! When explicitly setting the minimum macOS target to 10.9 on x86 in `bazel build`, proper 10.9 wheels are built. Thank you so much for your help!
